<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientMainWindow</class>
 <widget class="QMainWindow" name="clientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Collaborative Notepad 2.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="mainMenu">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QToolButton" name="lflbRefresh">
         <property name="text">
          <string>Local Files</string>
         </property>
        </widget>
        <widget class="QListWidget" name="localFileList"/>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QToolButton" name="flbRefresh">
         <property name="text">
          <string>Server Files</string>
         </property>
        </widget>
        <widget class="QListWidget" name="fileList"/>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPushButton" name="localDelete">
          <property name="text">
           <string>Local Delete</string>
          </property>
         </widget>
         <widget class="QPushButton" name="flbDownload">
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPushButton" name="flbEdit">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
         <widget class="QPushButton" name="serverDelete">
          <property name="text">
           <string>Server Delete</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLineEdit" name="lineEdit"/>
        <widget class="QPushButton" name="flbNew">
         <property name="text">
          <string>New File</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSplitter" name="editFrame">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPlainTextEdit" name="textBox"/>
      <widget class="QPushButton" name="tbbReturn">
       <property name="text">
        <string>Save &amp;&amp; Return</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
